src/box/tuple.c

```diff
@@ -118,8 +113,51 @@ tuple_next(struct tuple_iterator *it)
 }
 
 char *
+tuple_extract_key_sequential_raw(const char *data, const char *data_end,
+		      const struct key_def *key_def, uint32_t *key_size)
+{
+	assert(key_def_is_sequential(key_def));
+	const char *field_start = data;
+	uint32_t bsize = mp_sizeof_array(key_def->part_count);
+
+	mp_decode_array(&field_start);
+	const char *field_end = field_start;
+
+	for (uint32_t i = 0;i < key_def->part_count; i++) {
+		mp_next(&field_end);
+	}
+	bsize += field_end - field_start;
+
+	assert(!data_end || (field_end - field_start <= data_end - data));
+	(void) data_end;
+
+	char *key = (char *) region_alloc(&fiber()->gc, bsize);
+	if (key == NULL) {
+		diag_set(OutOfMemory, bsize, "region",
+				"tuple_extract_key_raw_sequential");
```

Comment:
Same as above - move "tuple_extract_key_raw_sequential" under the opening bracket of `diag_set()`